docs/section22/Rule22-34.md

```diff
@@ -0,0 +1,45 @@
+
+# CHW&CW - Rule 22-34  
+
+**Schema Version:** 0.0.10 **Mandatory Rule:** True  
+**Rule ID:** 22-34  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water must only have no more than one CHW plant.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. get_primary_secondary_loops()
+
+**Applicability Checks:**  
+
+- **Check 1:** Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- Get number of primary loops in B-RMR: `num_loops = LEN(primary_secondary_loop_dictionary)`
+
+**Rule Assertion - Component:**
+
+- Case 1: If only one CHW primary loop is modeled: `if num_loops == 1: PASS`
+
+- Case 2: Else: `else: FAIL`
+
+**[Back](../_toc.md)**
+
+**Notes:**
+
+1. Treat all buildings under RMR as one building.
+2. "no more than one CHW plant" means just one CHW primary loop?
+3. It seems App. G has no requirement for number of secondary loops.
```

Comment:
I think there should be only one secondary loop - G3.1.3.10 mentions both as singular loops: "G3.1.3.10 Chilled-Water Pumps (Systems 7, 8, 11, 12, and 13). Chilled-water systems shall be modeled as primary/secondary systems with constant flow primary loop and variable-flow secondary loop."

Reply:
Updated.